node-engine/reference/cli/wpe/apps/create.mdx

```diff
@@ -75,6 +79,41 @@ Providing custom `domains` is optional. If you do want to connect a custom domai
 
 **NOTE: We currently only support one custom domain.**
 
+## Create From a zip file
+
+**Note:** The zip app flow only works with wpe CLI version 1.0.0 or greater.
+
+You can create an Atlas app from a `json` file configured for zip uploads. All requirements from the standard `wpe create` flow apply, with a few exceptions.
+
+To make your `json` file zip configurable, set the `"repo"` parameter to `"uploadable"`. Then, add a parameter named `"zip_file_path"` with the directory path to your zip file in your environment variables. 
```

Comment:
"with the directory path to your zip file in your environment variables" is not quite right. User doesn't have to change anything in environment variables when using the zip upload flow.